site/src/components/UserDropdownContent/UserDropdownContent.tsx

```diff
@@ -5,32 +5,45 @@ import MenuItem from "@material-ui/core/MenuItem"
 import { makeStyles } from "@material-ui/core/styles"
 import Typography from "@material-ui/core/Typography"
 import AccountIcon from "@material-ui/icons/AccountCircleOutlined"
+import BugIcon from "@material-ui/icons/BugReportOutlined"
+import LaunchIcon from "@material-ui/icons/Launch"
+import ChatIcon from "@material-ui/icons/Chat"
 import { FC } from "react"
 import { Link } from "react-router-dom"
 import * as TypesGen from "../../api/typesGenerated"
 import { navHeight } from "../../theme/constants"
 import { DocsIcon } from "../Icons/DocsIcon"
 import { LogoutIcon } from "../Icons/LogoutIcon"
 import { UserAvatar } from "../UserAvatar/UserAvatar"
+import Tooltip from "@material-ui/core/Tooltip"
```

Comment:
Does v2 have sorting preferences for frontend imports?  We might need a linting rule to help automate it.